content/tutorials/react-native-presence-publish-subscribe.textile

```diff
@@ -0,0 +1,539 @@
+---
+title: Build a realtime chat app with react native and ably
+section: tutorials
+index: 0
+---
+
+
+h2. Introduction
+
+Messaging apps are on the increase recently. The past few years have brought apps like WhatsApp, Telegram, Facebook Messenger, Slack e.t.c.
+People seem to prefer chat-based applications because they allow for real-time interaction. They also add a personal touch to the experience. 
+In this tutorial, you will learn how to build a real-time chat application using React Native and ably.
+
+React Native lets you build mobile apps using only JavaScript. It uses the same design as React, letting you compose a rich mobile UI from declarative components. To learn more about React Native, please visit "here":https://facebook.github.io/react-native/.
+
+Ably is a real-time data delivery platform providing developers everything they need to create, deliver and manage complex projects. Ably solves the hardest parts so they don’t have to. Our 100% data delivery guarantee offers unique peace of mind to stream data between apps, websites and servers anywhere on earth in milliseconds.
```

Comment:
Note our dictionary is `realtime` as opposed to `real-time`, unless you use it in the context of something being done in real time.

Reply:
We'll overwrite this intro bit anyway.

Reply:
Fixed this, thanks